build/build-go.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 set -eu
 eval $(go env | grep -e "GOHOSTOS" -e "GOHOSTARCH")
+echo $GOPATH
```

Comment:
if want to keep this add a tag so we know what the output is, like: echo "GOPATH: $GOPATH"

Reply:
This is removed.